slimCat/Utilities/Extensions/StringExtensions.cs

```diff
@@ -216,7 +216,14 @@ public static Gender ParseGender(this string input)
                     return Gender.Cuntboy;
 
                 default: // every other gender is parsed normally
-                    return input.ToEnum<Gender>();
+                    try
```

Comment:
It should be able to do `default(whatever)`. I always write my enums so that the first value is an empty one for a reason like this